### ISSUE TITLE
Spur at inferior pole of patella

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#11195]
**Preferred term label:**
Spur at inferior pole of patella

**Synonyms**
Sinding-Larsen-Johansson disease

**Definition (free text, please give PubMed ID)**
Calcification at inferior pole of patella

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Sinding-Larsen-Johansson syndrome

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Sinding-Larsen-Johansson -- not an expert, but it does not seem to be a bone spur, it is more of an apophysitis?
https://pmc.ncbi.nlm.nih.gov/articles/PMC11438430/

**Comment by MickeySegal** [comment]
I will ask the orthopedist

**Comment by MickeySegal** [comment]
The orthopedist replied:

This is a tough one. I agree that Sinding-Larsen-Johansson is more of an apophysitis type injury, but I purposely left the radiographic finding vague so that it could be used for other diagnoses as well. As an example, patellar tendinopathy (enthesophyte formation), degenerative patellofemoral osteoarthritis (osteophyte formation), and heterotopic ossification can also result in abnormal bone formation at the inferior pole of the patella. If you want it to be pathognomonic for Sinding-Larsen-Johansson, then changing to apophysitis would make sense and we can do a separate category for some of these other diagnoses. If we leave as is, I think adding "enthesophyte, osteophyte, apophysitis" under the other definitions would be helpful for search purposes.

**Comment by pnrobinson** [comment]
@MickeySegal 
"I purposely left the radiographic finding vague so that it could be used for other diagnoses as well." => This is a good strategy, therefore I have left the term as is.

- ID: HP:6001377
- label: Spur at inferior pole of patella
- parents: [Abnormal patella morphology (HP:0003045)](https://hpo.jax.org/app/browse/term/HP:0003045)
- pmid: PMID:39346060
- def: A bone spur (osteophyte), i.e., de novo bone formation at the lower edge of the patella (knee cap). Radiographically, it is visible as calcification at inferior pole of the patella.
- comment: This finding is characteristic of Sinding-Larsen-Johansson disease.
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/11195
- synonyms:
